interfaces/datastructure/Frame.h

```diff
@@ -185,6 +193,7 @@ class SOLARFRAMEWORK_API Frame {
 protected:
     Transform3Df                    m_pose;    
     SRef<Image>                     m_view;
+    SRef<Image>                     m_mask; // to store 2D image segmentation result
```

Comment:
I guess it could also be discussed whether it's not simply an Image, or a unique pointer to an Image, but I guess we can leave it like this for now, for consistency with m_view.